cmake/external/abseil-cpp.cmake

```diff
@@ -8,7 +8,7 @@ set(ABSL_PROPAGATE_CXX_STD 1)
 set(BUILD_TESTING 0)
 
 if(Patch_FOUND)
-  set(ABSL_PATCH_COMMAND ${Patch_EXECUTABLE} --ignore-whitespace -p1 < ${PROJECT_SOURCE_DIR}/patches/abseil/Fix_Nvidia_Build_Break.patch)
+  set(ABSL_PATCH_COMMAND ${Patch_EXECUTABLE} --binary --ignore-whitespace -p1 < ${PROJECT_SOURCE_DIR}/patches/abseil/Fix_Nvidia_Build_Break.patch)
```

Comment:
just curious. why is --binary added? how is it related to this pr?

Reply:
it failed our internal build for SNPE windows with the latest code. I don't know the difference with the Windows build environment here. But from the web site it could happen, and this is the fix.
http://gnuwin32.sourceforge.net/packages/patch.htm
On MS-Windows, the patchfile must be a text file, i.e. CR-LF must be used as line endings. A file with LF may give the error: "Assertion failed, hunk, file patch.c, line 343," unless the option '--binary' is given.

Reply:
"I don't know the difference with the Windows build environment here" They should be exactly the same. Though their pool names are different, the other things are exactly the same.